core-addon/DataCollection.js

```diff
@@ -18,15 +18,16 @@ module.exports = class DataCollection {
   /**
    * Sends an empty Ion ping with the provided info.
    *
+   * @param {String} rallyId
+   *        The id of the Rally platform.
    * @param {String} payloadType
    *        The type of the encrypted payload. This will define the
    *        `schemaName` of the ping.
-   *
    * @param {String} namespace
    *        The namespace to route the ping. This will define the
    *        `schemaNamespace` and `studyName` properties of the ping.
    */
-  async _sendEmptyPing(payloadType, namespace) {
+  async _sendEmptyPing(rallyId, payloadType, namespace) {
```

Comment:
I think it would, yes. I'd love to have some time to think about that though, and leave it out as a future improvement. I'd like to make sure there's no weird security risk.